.pre-commit-config.yaml

```diff
@@ -1,26 +1,38 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 21.6b0
+    rev: 22.3.0
     hooks:
       - id: black
-        args: [--line-length=80]
+        args: ["--line-length=88"]
         additional_dependencies: ['click==8.0.1']
```

Comment:
shouldn't it use click 8.1.0 like in CI?

Reply:
Good catch. I'll change it!